File: programming files/sum_even_fibonacci.cc
#include <iostream>

int main()
{
    //changed - AK
    int a = 0, b = 1.0;
    int sum_even = 0;
    //changed - AK
    while (b =< 4000000) {
        if (b % 2 == 0) {
            // Adds b to sum_even when b is even
            sum_even += b;
        }
        int temp = b;
        b = a + b;
        a = temp;
    }

    // Prints the answer
    std::cout << sum_even << std::endl;

    return 0;
}

// answer should be 4613732

File: programming files/sum_even_fibonacci.py
a, b = 0, 1
#removed semicolon from line 3
sum_even = 0
# Only sum fibonacci values up to 4,000,000
#changed <= to < 
while b < 4000000:
    #changed / to %
    if b % 2 == 0:
        sum_even += b
    #changed b, b to b, a+b
    a, b = b, a + b

print(sum_even)

# answer should be 4613732
